frontend/src/i18n/locales/nl/voters_votes_counts.json

```diff
@@ -18,7 +17,6 @@
   "voters_recounts": {
```

Comment:
Is this section still needed? I don't thing we still reference a translation for `voters_recounts`.